test/webpack-build.test.js

```diff
@@ -193,18 +193,37 @@ test('Bundles can use custom names', async (test) => {
   test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'lambda', 'service.js')));
 });
 
-test('Multi-entry bundling cannot be used with bundle zipping', async (test) => {
-  const options = {
+test('Bundles for multiple entries can be zipped', async (test) => {
+  await build({
     entrypoint: [
-      path.join(__dirname, 'fixtures', 'lambda_service.js'),
-      path.join(__dirname, 'fixtures', 'lambda_graphql.js')
+      path.join(__dirname, 'fixtures', 'lambda_graphql.js:graphql.js'),
+      path.join(__dirname, 'fixtures', 'lambda_service.js:lambda/service.js')
+    ],
+    outputPath: test.context.buildDirectory,
+    serviceName: 'test-service',
+    zip: true
+  });
+
+  test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'graphql.js.zip')));
+  test.true(await fs.pathExists(path.join(test.context.buildDirectory, 'lambda/service.js.zip')));
+});
+
+test('Expand input entrypoint directory into multiple entrypoints', async (test) => {
+  await build({
+    entrypoint: [
+      path.join(__dirname, 'fixtures/multi-lambdas')
     ],
     outputPath: test.context.buildDirectory,
     serviceName: 'test-service',
     zip: true
-  };
+  });
+
+  await fs.mkdirp(path.join(__dirname, 'fixtures/multi-lambdas/empty'));
```

Comment:
This seems like it should be a permanent part of the fixtures directory or this test should operate on a temporary directory that is specific to this test case. Otherwise, other test cases will not be able to predictably account for this state change.

Reply:
I can't checkin an empty directory and the test is specifically testing an empty directory.

Reply:
So a temporary work directory specific to the test case or global test setup then? I just want to avoid potentially unexpected test results in the future because one test case is changing the fixtures directory contents during execution (and while multiple tests are running in parallel).

Reply:
Fixed. I create an empty directory whose name contains a `uuid`. I then delete the directory in a `finally` clause. Due to the nature of this test, the empty directory has to be nested under the `./test/fixtures/multi-lambdas` directory.

Reply:
I don't think that really addresses the issue since other `multi-lambdas` tests will still see the directory as state. I think you would need to create a copy of the `multi-lambdas` directory somewhere else to really address the issue.